test/docker-tests/src/test/java/it/unibz/inf/ontop/docker/ParallelScenarioManifestTestUtils.java

```diff
@@ -51,6 +50,7 @@
 import java.util.jar.JarFile;
 
 public class ParallelScenarioManifestTestUtils {
+	private static final QueryLanguage SERQL_QUERY_LANGUAGE = QueryLanguage.valueOf("SERQL");
```

Comment:
SERQL will be removed in RDF4J 4. Getting the constant indirectly makes the codebase easier to compile with RDF4J 4 (albeit the tests relying on SERQL will not run).